architecture/adr-list/adr-005-genesis-parameters.md

```diff
@@ -0,0 +1,165 @@
+---
+description: This is the suggested template to be used for ADRs on the cheqd-node project.
+---
+
+# ADR 005: Genesis parameters
+
+## Status
+
+PROPOSED \| Not Implemented
+
+## Summary
+
+The aim of this document is to define genesis parameters that will be used in testnets and the mainnet.
+
+> Cosmos v0.42.5 parameters are described.
+
+## Context
+
+Genesis consists of Tendermint consensus engine parameters and Cosmos app-specific parameters.
+
+### Consensus parameters
+
+Proposed values:
+
+* block
+  * max\_bytes = `22020096` \(~22MB\)
+    * Cosmos hub: `200000` \(~200KB\)
+  * max\_gas = `-1` \(no gas limit\)
+    * Cosmos hub: `2000000`
+  * time\_iota\_ms = `1000`
+    * Cosmos hub: `1000`
+    * **Deprecated, unused**
+* evidence
+  * max\_age\_num\_blocks = `100000`
+    * Max age of evidence, in blocks. The basic formula for calculating this is: `MaxAgeDuration / {average block time}`.
```

Comment:
There's a `blockperyear` defined in the genesis parameters spreadsheet. Check whether these align